tests/documentation/test_files/DocumentationStyleBear/bad_file.py.test.correct

```diff
@@ -0,0 +1,12 @@
+def hello_there(x):
+    """
```

Comment:
alright I can rephrase a bit clearer :)

But currently I won't support both modes, this bear is currently more or less a demonstration of the inline-documentation capabilities of coala, and as this PR is here for a quite long time I want this merged, such functionality can be added later.